src/devices/bosch.ts

```diff
@@ -326,6 +327,66 @@ const boschExtend = {
             isModernExtend: true,
         };
     },
+    roomThermostat: (): ModernExtend => {
```

Comment:
Since both `roomThermostat` and `radiatorThermostat` are generic, I think this can be moved to `modernExtend.ts` under a combined `thermostat`

Reply:
I think it's fine to add it there, don't see any risks for now